app/src/main/java/com/firebase/uidemo/storage/ImageActivity.java

```diff
@@ -82,7 +83,7 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
         super.onActivityResult(requestCode, resultCode, data);
 
         if (requestCode == RC_CHOOSE_PHOTO) {
-            if (resultCode == RESULT_OK) {
+            if (resultCode == ResultCodes.OK) {
```

Comment:
This seems like an IntelliJ refactor bug, in this case I am not using AuthUI I want to see if the Camera returned Activity.RESULT_OK